bin/run.sh

```diff
@@ -17,6 +17,7 @@ function main {
     --log-junit "${output_dir%/}/${XML_RESULTS}" \
     --verbose \
     --no-configuration \
+    --do-not-cache-result \
```

Comment:
Previously was emitting a warning because it could not write a cache file becasue of the readonly file system.  Added flag to prevent attempt to write file.